web/themes/new_weather_theme/templates/block/block--views-block--weather-story-weather-story.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="tablet:grid-col tablet:margin-right-2">
+  {{ content }}
+</div>
```

Comment:
Ideally we wouldn't need these templates at all, but the Drupal UX doesn't allow colons in class names, so we can't simply set them there. More documentation coming on this issue, because it may be relevant government-wide for anyone using Drupal and USWDS.

See this [6-year-old bug report](https://www.drupal.org/project/drupal/issues/2916377). There's a patch available so hopefully that means it'll be fixed in a future Drupal 10 release. 🤞